test/src/api/expect/testExpectPresent.js

```diff
@@ -170,6 +170,28 @@ describe('expect.present', function() {
       });
     });
 
+    it('to not be present if element doesn\'t exist [PASSED]', function() {
+      this.client.api.globals.waitForConditionTimeout = 40;
+      this.client.api.globals.waitForConditionPollInterval = 20;
+
+      Nocks.elementNotFound()
+        .elementNotFound()
+        .elementNotFound();
+
+      let expect = this.client.api.expect.element('#weblogin').to.not.be.present;
+
+      return this.client.start(function() {
+        assert.strictEqual(expect.assertion.selector, '#weblogin');
+        assert.strictEqual(expect.assertion.negate, true);
+        assert.strictEqual(expect.assertion.passed, true);
+        assert.strictEqual(expect.assertion.expected, 'present');
```

Comment:
as this test is for negation 'to not be present', shouldn't it be
`assert.strictEqual(expect.assertion.expected, 'not present');`
?